integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -128,6 +113,37 @@ public void initialize(String name, Map<String, String> properties) {
     this.cluster = properties.getOrDefault(CLUSTER_PROPERTY, DEFAULT_CLUSTER);
   }
 
+  /**
+   * sets the auth token as default header which gets added to every request from ApiClient
+   * @param token
+   */
+  protected void setToken(String token) {
+    if (token.isEmpty()){
```

Comment:
```suggestion
    if (!token.isEmpty()){
```

Reply:
How does testing done section produce expected result with this bug?

Reply:
This setToken is intended to be used internal OpenHouse repo to setAuth token fetched from UGI.